docs/pages/guides/using-analytics.mdx

```diff
@@ -70,3 +70,10 @@ The following list provides common analytics providers that are available in the
   href="https://posthog.com/docs/libraries/react-native"
   Icon={BookOpen02Icon}
 />
+
+<BoxLink
+  title="Dreambase"
+  description="Turn your Supabase data into answers. Learn how to integrate Dreambase in a few clicks."
```

Comment:
I think there's room to improve the description, since it doesn't explain why one should check out the link. I'll follow-up in a separate PR.